src/scenes/Dashboard/ProgressReportsWidget/RichTextCell.tsx

```diff
@@ -0,0 +1,96 @@
+import { Box, Typography } from '@mui/material';
+import {
+  GridState,
+  GridRenderCellParams as RenderCellParams,
+  useGridSelector,
+} from '@mui/x-data-grid';
+import { extendSx, RichTextJson, StyleProps } from '~/common';
+import {
+  RenderFn,
+  RichTextRenderers,
+  RichTextView,
+  Text,
+} from '../../../components/RichText';
+import { ProgressReportsDataGridRowFragment as ProgressReport } from './progressReportsDataGridRow.graphql';
+
+type CellParams = RenderCellParams<ProgressReport, RichTextJson>;
+
+export const RichTextCell = ({
+  id,
+  value,
+  api,
+  sx,
+  className,
+}: Pick<CellParams, 'value' | 'id' | 'api'> & StyleProps) => {
+  const selectedRows = useGridSelector(
+    { current: api },
+    (state: GridState) => state.rowSelection
+  );
+  const isExpanded = selectedRows[0] === id;
```

Comment:
We are calling this `selectedRows` then treating it as if there will be only one element.  I'd recommend either calling this `selectedRow` or running .find on this array.

Reply:
We can't just rename it, it'll still be an array.
This logic goes hand-in-hand with https://github.com/SeedCompany/cord-field/blob/7d22d905654fe30af90cd8289b9ad9c79a538447/src/scenes/Dashboard/ProgressReportsWidget/ProgressReportsExpandedGrid.tsx#L85-L89
which only ever has one element.

If you feel strongly about this though we can switch it to an `.includes()`